test/tests.jl

```diff
@@ -145,7 +145,7 @@ end
             str = String(take!(buf))
             @test occursin("```julia", str)
             @test occursin("f(x)", str)
-            @test occursin("[`$(joinpath("DocStringExtensions", "test", "tests.jl"))", str)
+            @test occursin(joinpath("DocStringExtensions", "test", "tests.jl"), str)
```

Comment:
Apparently on Travis the package directories get a `.jl` (i.e. `/DocStringExtensions.jl/` instead of `/DocStringExtensions`).  On Travis the full `str` is:

````
```julia
f(x)
```

defined at [`/home/travis/build/JuliaDocs/DocStringExtensions.jl/test/tests.jl:14`](https://github.com/JuliaDocs/DocStringExtensions.jl/tree/561ce5b42a227cd034b376d4d045bc2c3fa69a7c//test/tests.jl#L14).
````

FWIW, this test should probably pass even if the package is cloned into a directory that is not called `DocStringExtensions` or some variation of that. So I think it's sufficient to just search for `joinpath("test", "tests.jl")`.